### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

**Comment by miltontony** [comment]
@alexmuller when you PR is ready, can you @ me (or anyone really) for a review?

**Comment by alexmuller** [comment]
@miltontony Thanks, I think this is ready to be reviewed!

File: docs/cheatsheet.rst [hunk 1]
```diff
@@ -126,28 +126,34 @@ The steps are as follows:
...
 ::
-  hub push
+
+    hub push
 
 6. Convert the issue into a Pull Request
 ::
-  hub pull-request -b develop -i 1
+
+    hub pull-request -b develop -i 1
```
Comment (miltontony) [inline docs/cheatsheet.rst]: The `-i <issue_num>` has been deprecated and doesn't work anymore sadly. I think we should remove this. The command doesn't fail, it just doesn't convert an issue to a PR anymore. It creates a new issue number.

Comment (miltontony) [reply docs/cheatsheet.rst]: Can you test and see if this still works?

**Review by miltontony — APPROVED** [review]
1 small comment, otherwise, all good 
👍

File: docs/cheatsheet.rst (hunk 1 above)
Comment (alexmuller) [reply docs/cheatsheet.rst]: Interesting, good catch! For me it still reuses the issue number as expected (https://github.com/alexmuller/alexmuller.github.io/pull/1) but outputs:

> Warning: Issue to pull request conversion is deprecated and might not work in the future.

So as you say I think it probably makes sense to not include - I've reworded that bit.

**Pull request merged**